index.js

```diff
@@ -166,22 +165,21 @@ class HandlebarsRenderer {
      * @param {Object} A set of templates to register with handlebars
      */
     addTemplates(templates) {
-        _.each(templates, (template, path) => {
-            // Don't do this work twice, first one wins.
+        const paths = Object.keys(templates);
+
+        for (let i = 0; i < paths.length; i++) {
+            const path = paths[i];
+            let template;
```

Comment:
I would move this closer to its usage, i.e. right below the comment `// Check if it is a precompiled template`

Reply:
fixed